src/server/services/procedures/FBICrimeData/FBICrimeData.js

```diff
@@ -0,0 +1,364 @@
+    if (endYear<=startYear) {
+        throw Error('End year should not be less than or equal to Start year');
+    } 
+    else {
+        return await this._requestData({path:`api/arrest/regions/${regionName}/${offense}/${category}/${startYear}/${endYear}`, queryString:`api_key=${this.apiKey.value}`});
+    }
+};
+
+/**
+ * Gets the number of arrests(for a particular offense) for the state in a certain time period
+ * @param {String} stateAbbr location of the state the crime occurred
+ * @param {Enum<violent_crime,propery_crime,drug,human_trafficking,gambling,juvenile,society,all>} category variable affecting crimes including examples: count, weapons, etc.
+ * @param {BoundedInteger<1985, 2019>} startYear beginning year
+ * @param {BoundedInteger<1985, 2019>} endYear ending year
+ * @returns {Object} structured data representing the location of the address
+ */
+Crime.stateArrestCount = async function (stateAbbr, category, startYear, endYear) {
+    if (endYear<=startYear) {
+        throw Error('End year should not be less than or equal to Start year');
+    } 
+    else {
+        return await this._requestData({path:`api/arrest/states/offense/${stateAbbr}/${category}/${startYear}/${endYear}`, queryString:`api_key=${this.apiKey.value}`});
+    }
+};
+
+/**
+ * Gets the number of victims for the nation based on the offense and variable
+ * @param {OffenseVictim} offense the type of breach of a law or rule
+ * @param {Enum<age,count,ethnicity,race,sex,relationship>} category variable affecting crimes including examples: count, weapons, etc.
+ * @returns {Object} structured data representing the location of the address
+ */
+Crime.nationalVictimCount = async function (offense, category) {
+    return await this._requestData({path:`api/nibrs/${offense}/victim/national/${category}`, queryString:`api_key=${this.apiKey.value}`});
+};
+
+/**
+ * Gets the number of victims for the nation based on the offense and variable
+ * @param {OffenseVictim} offense the type of breach of a law or rule
+ * @param {String} regionName location of the region the crime occurred
+ * @param {Enum<age,count,ethnicity,race,sex,relationship>} category variable affecting crimes including examples: count, weapons, etc.
+ * @returns {Object} structured data representing the location of the address
+ */
+Crime.regionalVictimCount = async function (offense, regionName, category) {
+    return await this._requestData({path:`api/nibrs/${offense}/victim/regions/${regionName}/${category}`, queryString:`api_key=${this.apiKey.value}`});
+};
+
+/**
+ * Gets the number of victims for the nation based on the offense and variable
+ * @param {OffenseVictim} offense the type of breach of a law or rule
+ * @param {String} stateAbbr state the crime occurred
+ * @param {Enum<age,count,ethnicity,race,sex,relationship>} category variable affecting crimes including examples: count, weapons, etc.
+ * @returns {Object} structured data representing the location of the address
```

Comment:
Looks like all the return value descriptions are the same. You might need to test out each rpc and write the description based on whatever it gives you back. This also goes for rpc descriptions, which are mostly just slightly different. Feel free to make rpc descriptions long and informative if you think it would be helpful for students - I try to write things like [this](https://github.com/NetsBlox/NetsBlox/blob/515d81aff5e11ef094ddc7a75d8b66acae73b66a/src/server/services/procedures/phone-iot/phone-iot.js#L291-L312), which more or less explain every part in sufficient detail that students know what it does.